### ISSUE TITLE
[Notifier, Achievements] Use consistent letter case in notifications

### ISSUE THREAD

**Issue by psdtwk** [creesch/reddit-moderator-toolbox#517]
Notifications have a mixmash of upper and lower case letters. Let's make them consistent by using [sentence case](
http://en.wikipedia.org/wiki/Letter_case#Sentence_case).

**Comment by agentlame** [comment]
Eh... I donno if sentence case really makes sense to be universally used.  Both the name of the achievement and the title of a notification are by definition titles.

**Comment by agentlame** [comment]
Also, for the love of fucking fuck, is it toolbox or Toolbox?

**Comment by psdtwk** [comment]
> I donno if sentence case really makes sense to be universally used. Both the name of the achievement and the title of a notification are by definition titles.

I agree with you. Title Case for the Title, and sentence case for the body?

**Comment by creesch** [comment]
I am not even sure this warrants an issue, ThIs ObseSsion  witH upper and lower Cases is unhealthy.

**Comment by agentlame** [comment]
It's just a reminder to do check it out.  It's not in 3.1.
